samples/tab-stage-view/csharp/README.md

```diff
@@ -124,9 +124,9 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 
 ![Card Actions](TabInStageView/Images/cardActions.png)
 
-- Opening stage view from Adaptive Card Action.
+- Opening Collaborative Stage View. Please refer [Collaborative Stage view](https://review.learn.microsoft.com/en-us/microsoftteams/platform/tabs/tabs-link-unfurling?branch=pr-en-us-7891#collaborative-stage-view) for more details
```

Comment:
The link is a temporary link. Is that intended?

Reply:
Yes, once the docs are published we will update the link